packages/jellyfish-api-core/__tests__/category/oracle/listLatestRawPrices.test.ts

```diff
@@ -0,0 +1,255 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+import { OracleRawPriceState } from '../../../src/category/oracle'
+
+describe('Oracle', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should listLatestRawPrices', async () => {
+    const priceFeed1 = [
```

Comment:
```suggestion
    const APPLE = [
```
QOL improvement, I think name with asset symbols ease reading a lot.